pkg/config/audit/audit_policies.go

```diff
@@ -0,0 +1,29 @@
+package audit
```

Comment:
don't we have some apiserver specific package where we put this audit package under?

Reply:
ok, will move to `pkg/operator/apiserver/audit`